src/components/editor/ArticleListController.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
```

Comment:
As an overall note. I would rename this to `EditorArticleListController`, it's too easy to open the wrong file if you share names with the client components. Also unlikely, but possible that you would want to import the Client controller and Editor one into the same file and you will get a namespace conflict.